Utilities/rXml.cpp

```diff
@@ -58,7 +71,7 @@ std::shared_ptr<rXmlNode> rXmlNode::GetChildren()
 
 std::shared_ptr<rXmlNode> rXmlNode::GetNext()
 {
-	wxXmlNode* result = reinterpret_cast<wxXmlNode*>(handle)->GetNext();
+	pugi::xml_node* result = &(reinterpret_cast<pugi::xml_node*>(handle)->next_sibling());
```

Comment:
[-Waddress-of-temporary]
  ...result = &(reinterpret_cast<pugi::xml_node*>(handle)->next_sibling());

Not sure what this is supposed to be doing but it ain't right..